docs/source/architecture/transactions_and_batches.rst

```diff
@@ -132,10 +149,10 @@ message types:
    :caption: File: protos/batch.proto
    :linenos:
 
-Header, Signature, and Public Keys
-----------------------------------
+Batch Header, Signature, and Public Keys
+----------------------------------------
 
-Following the pattern presented in Transaction, the Batch header field is a
+Following the pattern presented in a Transaction, the Batch header field is a
```

Comment:
the article is actually "the definition of the Transaction structure", not "an instance of Transaction"

Reply:
Thanks.  Rewrote to clarify: "the Transaction data structure".